File: src/client/classes/api/req_res/get-ur-log-response.ts
'use strict'

import { URLog } from '@/classes/datas/ur-log'
import { GkillAPIResponse } from '../gkill-api-response'

export class GetURLogResponse extends GkillAPIResponse {

    urlog_histories: Array<URLog>

    constructor() {
        super()
        this.urlog_histories = new Array<URLog>()
    }

}

File: src/server/gkill/api/req_res/get_ur_log_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type GetURLogResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	URLogHistories []reps.URLog `json:"urlog_histories"`
}
